src/Accelerators/InitAccelerators.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===------------------------ InitAccelerators.cpp ------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Initialization of accelerators.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Accelerators/Accelerator.hpp"
+
+namespace onnx_mlir {
+namespace accel {
+
+//
+bool initAccelerators() {
```

Comment:
Is there a reason this is not capitalized? I know it was added in a previous change but it seems weird since everything else is capitalized